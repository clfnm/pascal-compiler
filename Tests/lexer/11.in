program Test; 
const C=10; 
type TCounter=0..C; 
var i:TCounter; 
CurEl:integer; 
summa:integer; 
//begin writeln (‘Hello World!!!’); i:=0; summa:=0; repeat read(CurEl); summa:=summa+CurEl; i:=i+1; until i=C; write(summa); end.
begin writeln (‘Hello World!!!’); 
i:=0; 
summa:=0; 
repeat read(CurEl); 
summa:=summa+CurEl; 
i:=i+1; 
until i=C; 
write(summa); 
end.