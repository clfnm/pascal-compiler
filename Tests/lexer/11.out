1	1	Separator	Dot	.
1	2	Eof	EOF	